src/common/BaseLightAccount.sol

```diff
@@ -21,14 +21,10 @@ abstract contract BaseLightAccount is BaseAccount, TokenCallbackHandler, UUPSUpg
         CONTRACT_WITH_ADDR
     }
 
-    /// @dev The length of the array does not match the expected length.
```

Comment:
These were self-explanatory, so removing to be consistent with errors in BaseLightAccountFactory.